src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerSelector.iOS.cs

```diff
@@ -219,6 +219,14 @@ private void UpdatePickerStyle()
 				return;
 			}
 
+			//iOS 15 brought back the Dial Wheels for the Date Picker
```

Comment:
Would it make sense to extract `UpdatePickerStyle` in a separate helper class (either as a regular static method or as an extension method) so that its code isn't duplicated in both classes?

Reply:
I think that for a small portion of code like this one, it's not particularly important.